packages/schematics/src/utils/common.ts

```diff
@@ -24,7 +24,7 @@ export function addUpgradeToPackageJson(): Rule {
 }
 
 export function offsetFromRoot(fullPathToSourceDir: string): string {
-  const parts = fullPathToSourceDir.split('/');
+  const parts = fullPathToSourceDir.split('/').filter(part => !!part);
```

Comment:
are you filtering our // or a trailing /? add a comment explaining why you are filtering it out